pkg/app/app.go

```diff
@@ -27,8 +28,14 @@ type App struct {
 	FpsCounterText *ui.Text
 }
 
-func InitApp(windowWidth int, windowHeight int, windowCentered bool, windowTitle string) *App {
+func init() {
```

Comment:
Had to move the init code in the package init to make sure it's executed before inits of other packages